code/modules/unit_tests/strange_reagent.dm

```diff
@@ -97,6 +97,8 @@
 /datum/unit_test/strange_reagent/proc/test_death_no_damage(target_type)
 	var/mob/living/target = allocate_new_target(target_type)
 	target.death()
+	if(QDELETED(target))
+		return
```

Comment:
no, this should not be needed. they should not be qdel'd here

Reply:
there's also the fact that there should not be any bandaid code in unit tests

Reply:
mobs can gib in death, gibbing qdels

Reply:
I forgot how much I over engineered this test